app/scheduler/modules/pairing.py

```diff
@@ -0,0 +1,119 @@
+"""Match two Launch Pad member for a private conversation"""
+from slack import WebClient
+from interface.slack import Bot
+from random import shuffle
+from .base import ModuleBase
+from typing import Dict, List, Any, Set
+from flask import Flask
+from config import Config
+from db.facade import DBFacade
+from app.model import Pairing
+import logging
+
+
+class PairingSchedule(ModuleBase):
+    """Module that pairs members each ``SLACK_PAIRING_FREQUENCY``"""
+
+    NAME = 'Pair members randomly for meetups'
+
+    def __init__(self,
+                 flask_app: Flask,
+                 config: Config,
+                 facade: DBFacade):
+        """Initialize the object."""
+        self.bot = Bot(WebClient(config.slack_api_token),
+                       config.slack_notification_channel)
+        self.channel_id = self.bot.get_channel_id(config.slack_pairing_channel)
+        self.config = config
+        self.facade = facade
+
+    def get_job_args(self) -> Dict[str, Any]:
+        """Get job configuration arguments for apscheduler."""
+        logging.info(f"Running pairing at cron job: \
+            {self.config.slack_pairing_frequency}")
+        cron_frequency = self.config.slack_pairing_frequency.split(' ')
+        return {'trigger':      'cron',
+                'minute':        cron_frequency[0],
+                'hour':         cron_frequency[1],
+                'day':          cron_frequency[2],
+                'month':         cron_frequency[3],
+                'day_of_week':   cron_frequency[4],
+                'name':          self.NAME}
+
+    def do_it(self):
+        """Pair users together, and create a private chat for them"""
+        users = self.bot.get_channel_users(self.channel_id)
```

Comment:
Probably want to stop execution when we find out that there are no users in the channel (it also removes a check in `__pair_users()` that checks to see if there are any users in the list).